programmers/예산.js

```diff
@@ -1,7 +1,6 @@
 function solution(d, budget) {
     
     d.sort((a, b) => {return a - b;});
```

Comment:
```suggestion
    d.sort((a, b) => a - b);
```
return에 하나의 식만 있다면 return을 생략할 수 있어요